dev.c

```diff
@@ -210,14 +210,17 @@ static void request_end(struct fuse_conn *fc, struct fuse_req *req,
 	int status)
 {
 	u64 uid = req->in.unique;
+#ifdef __PX_BLKMQ__
+	bool using_blkque = req->pxd_dev->using_blkque;
+#endif
 	if (req->end)
 		req->end(fc, req, status);
```

Comment:
after this call if blkmq block device was used, fuse req (req) is no longer available. crash below.